handlers/user.go

```diff
@@ -1181,6 +1181,13 @@ func UserDeletePost(w http.ResponseWriter, r *http.Request) {
 		}
 
 		Logout(w, r)
+		err = purgeAllSessionsForUser(r.Context(), user.UserID)
+		if err != nil {
+			utils.LogError(err, fmt.Errorf("error purging sessions for user %v", user.UserID), 0)
```

Comment:
Parameter should always be separated from text: `utils.LogError(err, "error purging sessions for user", 0, map[string]interface{}{"userID":  user.UserID})`